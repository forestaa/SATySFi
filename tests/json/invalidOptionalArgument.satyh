let-inline \f x = {y} in {\f?:(3){3}}
