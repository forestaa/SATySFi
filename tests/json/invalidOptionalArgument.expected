{
  "file": "invalidoptionalargument.satyh",
  "range": {
    "start": { "line": 1, "character": 28 },
    "end": { "line": 1, "character": 33 }
  },
  "message":
    "! [Type Error] invalid application of an optional argument;\n    the command has type\n
['a] inline-cmd."
}
